src/main/java/com/launchdarkly/eventsource/EventParser.java

```diff
@@ -43,7 +43,10 @@ public void line(String line) {
       processComment(line.substring(1).trim());
     } else if ((colonIndex = line.indexOf(":")) != -1) {
       String field = line.substring(0, colonIndex);
-      String value = line.substring(colonIndex + 1).replaceFirst(" ", EMPTY_STRING);
+      String value = line.substring(colonIndex + 1);
+      if (value.charAt(0)==' ') {
```

Comment:
Note for posterity's sake: this is the correct behavior according to the spec:

https://www.w3.org/TR/eventsource/#parsing-an-event-stream